src/tests/operate.test.js

```diff
@@ -0,0 +1,56 @@
+import calculate from '../logic/calculate';
+import '@testing-library/jest-dom';
+
+describe('operate function check', () => {
+  test('AC', () => {
+    const obj = {
+      total: '50',
+      next: '2',
+      operation: '+',
+    };
+    const state = calculate(obj, 'AC');
+    expect(state.total).toBe(null);
+  });
+
+  test('number', () => {
+    const obj = {
+      total: '0',
+      next: null,
+      operation: '+',
+    };
+    const state = calculate(obj, '1');
+    expect(state.next).toBe('1');
+  });
+
+  test('dot', () => {
+    const obj = {
+      total: '3',
+      next: null,
+      operation: null,
+    };
+    const state = calculate(obj, '.');
+    expect(state.next).toBe('3.');
+  });
+
+  test('equal', () => {
+    const obj = {
+      total: '50',
+      next: '2',
+      operation: '+',
+    };
+
+    const state = calculate(obj, '=');
+    expect(state.next).toBe(null);
+  });
+
+  test('+/-', () => {
+    const obj = {
+      total: '50',
+      next: null,
+      operation: null,
+    };
+
+    const state = calculate(obj, '+/-');
+    expect(state.total).toBe('-50');
+  });
+});
```

Comment:
- [ ] Good job implementing tests 👍 However, please make sure to write tests for `operate.js` file. I noticed that in `operate.test.js`, you are writing tests for `calculate.js`. Please make sure to avoid repeating the tests and implement unit tests for the `operate.js` file using Jest as given in the project requirements ✨